backend/sirius-components-emf/src/main/java/org/eclipse/sirius/components/emf/view/OperationInterpreter.java

```diff
@@ -54,11 +54,11 @@
 import org.eclipse.sirius.ecore.extender.business.internal.accessor.ecore.EcoreIntrinsicExtender;
 
 /**
- * Executes the body of a tool as defined by a set of {@link Operation}s.
+ * Executes the body of a tool as defined by a set of {@link Operation}s or a set of {@link Operation}s.
  *
  * @author pcdavid
  */
-public class ToolInterpreter {
+public class OperationInterpreter {
```

Comment:
I'm not 100% sure that mixing in the same place concerns relative to both Diagrams and Forms is a good idea. Can't the diagram specific parts be extracted? We will talk about that (maybe tomorrow)

Reply:
splitted into two Interpreter.